### ISSUE TITLE
user docs 8.7 remotes provide examples

### ISSUE THREAD

**Issue by JMante1** [SynBioHub/synbiohub.github.io#16]
There are things listed that are not immediately obvious and should have examples:
- what is the remote id, give an example
- what is the remote url
- what is a sequence suffix and how is it used?

This section also needs more of an explanation of what a remote is and how it is integrated into synbiohub. what additional feature does adding a remote provide?

**Comment by PRUBHTEJ** [comment]
I do not have much knowledge about remotes, to be honest. I would request Prof. @cjmyers  to throw some light on this.

**Comment by PRUBHTEJ** [comment]
Since this was not mentioned in the API docs as well, so I don't have much knowledge about it

**Comment by JMante1** [comment]
@cjmyers have you seen this issue?

**Comment by cjmyers** [comment]
Add to beginning:

"SynBioHub is capable of connecting to remote repositories to allow data to be transferred to and from these remote repositories."

**Comment by cjmyers** [comment]
2nd table uses Benchling where you should use ICE.

**Comment by PRUBHTEJ** [comment]
We still don't have definitions for ICE API Token Client, ICE API Token Owner and Group ID.

**Comment by cjmyers** [comment]
Token Client - Domain of the client using the ICE API
Token Owner - Email address of the client using the ICE API
Group ID - ID number for the ICE group

**Comment by PRUBHTEJ** [comment]
> 2nd table uses Benchling where you should use ICE.

I did not get this. Could you please be more specific?

**Comment by PRUBHTEJ** [comment]
I've added the respective definitions.

**Comment by cjmyers** [comment]
Looks good to me.  @JMante1 ?

**Comment by PRUBHTEJ** [comment]
@JMante1 Please have a look.

**Comment by JMante1** [comment]
I would still like to see examples.

**Comment by PRUBHTEJ** [comment]
Could you please explain in a bit more depth?
How can I add examples?

**Comment by JMante1** [comment]
@cjmyers can you provide these?

**Comment by cjmyers** [comment]
id - acs
type - ice 
url - https://acs-registry.jbei.org
sequence suffix - _sequence
default folder ID - ace_ice_folder
folder prefix - ice_folder_
root collection display id - acs_collection
root collection name - ACS Synthetic Biology
root collection description - ICE repository for ACS Synthetic Biology Journal
ICE API Token - <string token>
ICE API Token Client - synbiohub.org
ICE API Token Owner - <email address>
ICE Collection - FEATURED
Group ID - 4
PI - <name>
PI Email - <email>
Part Number Prefix - acs

**Comment by cjmyers** [comment]
For benchling, differences are:
id - benchling
type - benchling
URL - https://api.benchling.com/v1
sequence suffix - _sequence
Default folder id - lib_LHwAaloE
folder prefix - folder_
root collection display id - benchling_collection
root collection name - Benchling
root collection description - Benchling collection of parts
benchling api token - <token string>

**Comment by PRUBHTEJ** [comment]
I have tried to do the required changes, @cjmyers @JMante1. Kindly have a look.

**Comment by cjmyers** [comment]
I suggest putting the examples into the table like this:

1. ID    Enter ID of the benchling remote (ex. benchling)

**Comment by PRUBHTEJ** [comment]
Understood, Prof. Myers. Since we already have a table, should. I add an additional column for examples?
Would that be alright since having another table might feel redundant.

<img width="902" alt="Screenshot 2022-03-28 at 11 36 57 PM" src="https://user-images.githubusercontent.com/47414674/160459773-8ce0b36b-c50f-402d-a9ba-b5f4b3495caa.png">

**Comment by cjmyers** [comment]
Yes, my suggestion was to include it in the same table.  You don't need an extra column, but just include it in the Description column like this:

<description> (ex. <example>)

**Comment by PRUBHTEJ** [comment]
Understood. I'll do it ASAP.
